Game/graphics/mdlvisual.cpp

```diff
@@ -491,15 +491,24 @@ Tempest::Vec3 MdlVisual::displayPosition() const {
   return {0.f,0.f,0.f};
   }
 
-Vec3 MdlVisual::lookAtVector(const char *nodeName) const
+bool MdlVisual::boneRotationY(const char *nodeName, float& deg) const
 {
-  if(nullptr==skeleton) {
-    return Vec3(0.f, 0.f, 0.f);
-  }
-  size_t nodeId = skeleton->findNode(nodeName);
-  Matrix4x4 trafo = skInst->tr[nodeId];
-  Vec3 atVector = Vec3(trafo.at(0,0),trafo.at(0,1),trafo.at(0,2));
-  return atVector;
+  size_t nodeId = size_t(-1);
+
+  if(nullptr!=skeleton)
+    nodeId = skeleton->findNode(nodeName);
+
+  if(size_t(-1)!=nodeId) {
+    auto p = pos;
+    p.mul(pose().tr[nodeId]);
+    float atVecX = p.at(2,0);
```

Comment:
Interesting, too:
If I change this to 
```cpp
x = p.at(0,0)
z = p.at(0,2)
```
which should be the right vector, I get a difference around 90° for all NPCs (including the one with the bench animation).